lib/routes/one/news.ts

```diff
@@ -0,0 +1,87 @@
+import { Route, Data, DataItem } from '@/types';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+
+const apiUrl = 'https://wufazhuce.com/';
+const NAME = '「ONE · 一个」';
+
+export const route: Route = {
+    path: `/news`,
```

Comment:
```suggestion
    path: '/news',
```
[Use single quotes](https://docs.rsshub.app/joinus/advanced/script-standard#string)